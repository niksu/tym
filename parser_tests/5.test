% Example of recursive predicate.

a().
b() :- c().
b() :- b().

