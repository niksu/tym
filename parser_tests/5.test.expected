input contents |% Example of recursive predicate.

a().
b() :- c().
b() :- b().

|
(no query given)
stringed file contents (size=79, remaining=1921)
|predicate_a().
predicate_b() :- predicate_c().
predicate_b() :- predicate_b().|
stringed file contents strlen=78
